recipes-bsp/bootfiles/rpi-bootfiles.bb

```diff
@@ -5,14 +5,14 @@ LIC_FILES_CHKSUM = "file://LICENCE.broadcom;md5=c403841ff2837657b2ed8e5bb474ac8d
 
 inherit deploy nopackages
 
-RPIFW_DATE ?= "20240319"
-SRCREV = "9f24f4bc2bdd07ffd158cfbb4bce88a2efc4c1f5"
+RPIFW_DATE ?= "20250415"
+SRCREV = "a2c7e696fa33727d79116c205844066e0099f66c"
 SHORTREV = "${@d.getVar("SRCREV", False).__str__()[:7]}"
-RPIFW_SRC_URI ?= "https://api.github.com/repos/raspberrypi/firmware/tarball/9f24f4bc2bdd07ffd158cfbb4bce88a2efc4c1f5;downloadfilename=raspberrypi-firmware-${SHORTREV}.tar.gz"
+RPIFW_SRC_URI ?= "https://api.github.com/repos/raspberrypi/firmware/tarball/a2c7e696fa33727d79116c205844066e0099f66c;downloadfilename=raspberrypi-firmware-${SHORTREV}.tar.gz"
```

Comment:
Shouldnt SRCREV variable be used here?

Reply:
That's right. I have changed it like this